### PR TITLE
feat: added wishlist and cart page

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Wishlist and Cart Page**
> added quantity counter component
> refactored products page
> fixed navbar

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]

**Comment by Richa-rani02** [comment]
code looks clean and well structured

File: src/pages/Wishlist/Wishlist.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <div className="wishlist_container">
+            <div className="head">Wishlist</div>
+            <div className="items">
+                Total Number of items: {products.length}
+            </div>
+            <div className="content">
+                {products.map((product) => (
+                    <Product
+                        key={product.id}
```
Comment (parthgabani777) [inline src/pages/Wishlist/Wishlist.jsx:17]: can use object destructuring

File: src/pages/Cart/Cart.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    </div>
+                ))}
+            </section>
+
+            <section className="order__summary">
+                <div className="title">Order Summary</div>
+
+                <ul className="summary">
+                    <li>
+                        Subtotal <span>₹ 28,185</span>
```
Comment (parthgabani777) [inline src/pages/Cart/Cart.jsx:67]: change price according to the item in the cart, currently it is only showing static price

Comment (nerdyspook) [reply src/pages/Cart/Cart.jsx:67]: Yes, I did this just to get a view of what my final page will look like. I will be refactoring it when I will add API for backend.

**Review by parthgabani777 — APPROVED** [review]

**Pull request merged**